package.json

```diff
@@ -24,18 +24,19 @@
       "!main.js"
     ],
     "mac": {
-      "artifactName": "${productName}-${version}-${arch}.${ext}",
+      "artifactName": "jitsi-meet-${version}-${arch}.${ext}",
```

Comment:
Why is this needed? Isn't that variable properly used?

Reply:
gotcha. can we drop the arch part from the macOS name? it's going to be x64 always anyway

Reply:
Yeah. I can remove that. But It is actually automatically being dropped. I guess we don't need to do that.